watershed-android/watershed/app/src/main/java/com/blueprint/watershed/MiniSites/MiniSiteFragment.java

```diff
@@ -94,6 +94,23 @@ public void onResponse(MiniSite miniSite) {
         mNetworkManager.getRequestQueue().add(request);
     }
 
+    private void setButtonListeners(View view){
+        View editButton = view.findViewById(R.id.minisite_edit_minisite);
+        editButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                mParentActivity.replaceFragment(EditMiniSiteFragment.newInstance(mSiteID, mMiniSite));
+            }
+        });
+        View miniSiteCreate = view.findViewById(R.id.minisite_add_fieldreport);
+        miniSiteCreate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                // Add Field Report. Unless we are requiring a user to be completing a task to submit a report. 
```

Comment:
what's gonna go in this onclicklistener if we're not allowing field reports without tasks?